_data/team.yml

```diff
@@ -10,7 +10,7 @@
   name: Mashooq Badar
   image: /assets/img/custom/team/mash_badar.jpg
   role: Software Craftsman and Co-Founder
-  fullDescription: <p>Mash is a software craftsman with extensive experience in developing innovative solutions and running large scale enterprise projects. For many years he was a Principal Consultant leading some of the most diverse projects for clients in Government, Media and Finance. Before Codurance Mash was a Director at UBS leading a 4 million pounds a year project with 6 development teams situated in London and Krakow. He started the project from scratch, building the teams a developer at a time.</p><p>Mash is an associated organiser for LSCC, involved since the beginning helping with organisation and running hands-on sessions. Mash has a passion for developing; teams that are skillful and passionate, solutions that are innovative and simple, and software that is aligned to it's business and flexible to change.</p>
+  fullDescription: <p>Mash is a pragmatic software craftsman always looking to improve his software creation skills and helping others do the same. He firmly believes that a well-rounded software craftsman must have a keen interest in all aspects of software creation, including; process, people, technology, user experience, development, operation, maintenance, and social impact. He relishes the daily challenges that Codurance brings to him–stretching his existing knowledge and expertise allowing him to constantly grow as a professional.</p><p>Mash is an advisor and a leader. During his diverse career he has succeeded in invigorating large ailing software projects as well as creating highly effective software teams and departments. His broad and deep technical knowledge, organisational skills, craft focus, and empathy to people involved have been integral to his success. He has worked in many roles for charities, investment banks, consultancies, government, media and cloud providers. He prides himself at being a hands-on software developer and believes that software development skills are very hard to learn and the best way to maintain them is to apply them.</p>
```

Comment:
- his diverse career he has succeeded ==> needs a comma ==> his diverse career, he has succeeded
- ailing project ---- did you mean --- failing project?
- at being ==> **on** being...
- Nit-picking, minor: I'm wondering if it is necessary to repeat _Mash_ in the bio, mentioning _Mash_ in the beginning - once might be enough and then use the pronoun _he_.